### PR TITLE
Remove CI jobs using non existing tags

### PR DESCRIPTION
**Pull request by zakkak**
Fixes recent (since the merge of https://github.com/oracle/graal/pull/6738)  CI failures, e.g.:

* https://github.com/oracle/graal/actions/runs/5176166290/jobs/9324638530
* https://github.com/oracle/graal/actions/runs/5177381458/jobs/9327406668

### CODE REVIEW

**Comment by zakkak** [comment]
For the record, the strict tag checking was added to `mx` with https://github.com/graalvm/mx/commit/b209e64b352788f4ae462a5263fff6aaac7d68bd

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -79,13 +79,9 @@ jobs:
               PRIMARY: "substratevm"
           - env:
               JDK_VERSION: "21"
-              GATE_TAGS: "build,test,helloworld"
```
Comment (zapster) [inline .github/workflows/main.yml:82]: I guess it makes sense to replace that with something else. This is what the internal CI is running:
https://github.com/oracle/graal/blob/master/substratevm/ci/ci.jsonnet#L115
So maybe run `native_unittests` here...

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -79,13 +79,9 @@ jobs:
...
+              GATE_TAGS: "build,helloworld"
               PRIMARY: "substratevm"
               PIP_PACKAGES: "jsonschema==4.6.1"
-          - env:
-              JDK_VERSION: "21"
-              GATE_TAGS: "build,helloworld_debug"
```
Comment (zapster) [inline .github/workflows/main.yml:87]: ... and `debuginfotest` here?

**Comment by zapster** [comment]
Thanks @zakkak for the fix. I guess the tags were originally added to test something but we did not notice they no longer exist (that is what motivates the strict tag checking in the first place). So we should probably replace them with something else. Some suggestions in-line.

File: .github/workflows/main.yml (hunk 1 above)
Comment (zakkak) [reply .github/workflows/main.yml:82]: Interesting, I didn't notice we were not running `native_unittests`... Adding it, thanks

File: .github/workflows/main.yml (hunk 2 above)
Comment (zakkak) [reply .github/workflows/main.yml:87]: We already run `debuginfotest` in https://github.com/oracle/graal/blob/f658f09475f468ed9e50934267c44aa6e4c4987a/.github/workflows/main.yml#L85-L88

**Review by zapster — APPROVED** [review]
Thanks again. I'll integrate it ASAP.

**Pull request merged**